recipes/gauxc/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "gauxc" %}
+{% set version = "1.0" %}
+{% set build = 0 %}
+{% set mpi = mpi or "nompi" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/wavefunction91/gauxc/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: dee246c8329cf48be4b2938776016196076e4c998ca39ee5f2e35eaac9569e68
+    patches:
+      - patches/gh-152.patch  # Allow building with Clang 19
+      - patches/gh-156.patch  # Allow building with Apple Clang
+      - patches/gh-160.patch  # Allow building shared libs
+      - patches/gh-161.patch  # Allow finding highfive library
```

Comment:
```suggestion
```
If the upstream author @wavefunction91 is able to do so, could this not just be rolled into a new tagged release (even just `1.0.1`)?  That would save us the trouble of having to remove the patches on the next release when they will no longer work.